Onion.Core/Interfaces/Services/ILoggerService.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Onion.Core.Models;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Onion.Core.Interfaces.Services
+{
+    public interface ILoggerService
```

Comment:
This feels a little funny. We have an ILoggerService interface. Which says to me that we've created an application service to pull together multiple things. So I anticipated something that pulled together an error handler and logger for example. In this case it looks like we're getting something? Maybe just a rename here... unsure of the purpose immediately.

Reply:
Yes, the principal idea here is to get information than was append in a log file.

We need to get whats is happening in the log file without the necessity to go into de server and download the file logger. 

I think we can change the name and the return type for:

`LogInfo GetInfoById(Guid Id)`

This method its going call it after the user received an email and clicked on the link

Reply:
Sounds great.